test/properties_test.dart

```diff
@@ -17,6 +19,26 @@ void main() {
     test('Existing by name', () => expect(Properties(baseFile), isNotNull));
   });
 
+  group('Save to file', () {
+    late Properties p;
+    late Properties sp;
+    setUp(() {
+      p = Properties.fromFile(baseFile);
+      p.saveToFile(saveFile);
+      sp = Properties.fromFile(saveFile);
```

Comment:
We should remove If the file first before calling saveToFile() to ensure the test is is actually testing the output of whatever was written right now, not in a past test run.

Additionally, I'd say the temporary file should be removed after the test ends (whether it passed or failed) & not gitignored.